sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -171,6 +171,18 @@ private[sql] class SharedState(
 
 object SharedState extends Logging {
   @volatile private var fsUrlStreamHandlerFactoryInitialized = false
+  private var sharedState: SharedState = _
```

Comment:
also nit: I think this variable name should be prefixed with `_` as in `_sharedState`. that would make it consistent with how other classes handle internal state: https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/SparkContext.scala#L202

Reply:
updated